### ISSUE TITLE
[Refactor] Forms, Fieldsets, Legends should be set properly

### ISSUE THREAD

**Issue by shindigira** [cfpb/sbl-frontend#422]
Upcoming Changes:
- `FormWrapper` should be renamed to `PageLayoutWrapper`
- `form` should be higher up on pages encompassing the `TextIntroduction`
- `fieldset` should be used to surrounded groups of  interactive controls.

**Comment by billhimmelsbach** [comment]
Yep @shindigira, I have work on that. We can discuss the breakdown of this maybe later today.

**Comment by billhimmelsbach** [comment]
There's still some lingering commits that need to be ported over.
